versioned_docs/version-0.1.8/functions/function-crd.md

```diff
@@ -84,32 +82,24 @@ When you specify a function or connector, you can optionally specify how much of
 
 If the node where a Pod is running has enough of a resource available, it's possible (and allowed) for a pod to use more resources than its `request` for that resource specifies. However, a pod is not allowed to use more than its resource `limit`.
 
-## Secrets
-
-In Function Mesh, the secret is defined through a secretsMap. To use a secret, a Pod needs to reference the secret. Pods can consume secretsMaps as environment variables in a volume. You can specify the `data` field when creating a configuration file for a secret. 
-
-To use a secret in an environment variable in a Pod, follow these steps.
-
-1. Create a secret or use an existing one. Multiple Pods can reference the same secret.
-2. Modify your Pod definition in each container, which you want to consume the value of a secret key, to add an environment variable for each secret key that you want to consume.
-3. Modify your image and/or command line so that the program looks for values in the specified environment variables.
+## Authentication
 
-Pulsar clusters supports using TLS or other authentication plugin for authentication.
+Function Mesh provides the `TLSSecret` and `AuthSecret` fields for Function, Source and Sink in the CRD definition. You can configure TLS encryption and/or TLS authentication using the following configurations.
```

Comment:
need a comma after "Source" in the first sentence ("...Function, Source, and Sink..")